dash/dash.py

```diff
@@ -1981,9 +1981,9 @@ def delete_resource(resources):
 
     def run(
         self,
-        host=os.getenv("HOST", "127.0.0.1"),
-        port=os.getenv("PORT", "8050"),
-        proxy=os.getenv("DASH_PROXY", None),
+        host=None,
+        port=None,
```

Comment:
```suggestion
        host="127.0.0.1",
        port="8050",
```

Reply:
I was planning to ask the same, because they were going to be overwritten anyway, and defaults were already set. Will make the changes.